tools/ansible/tasks/wsu/main.yaml

```diff
@@ -109,8 +111,18 @@
       retries: 3
       delay: 5
 
+    # Check if worker label is already applied to the Windows nodes bootstrapped via WMCB
+    # We should always get a single node that we've bootstrapped recently, if not skip this
+    # step
+    - name: Check if worker label is already applied to the Windows nodes bootstrapped via WMCB
+      delegate_to: localhost
+      shell: "oc get node {{ node_name.stdout_lines[0] }} -o yaml | grep -i node-role.kubernetes.io/worker"
+      ignore_errors: yes
```

Comment:
What errors are we ignoring?

Reply:
And is it safe to ignore all errors?

Reply:
We'll just get one error out of the shell command when the label is not present, which we can ignore.